src/internal/modules/billing-accounts/pre-handlers.js

```diff
@@ -85,7 +85,7 @@ const isRebillableBill = bill =>
   (bill.batch.source === 'wrls') &&
   !bill.isDeMinimis &&
   (bill.netTotal !== 0) &&
-  isEmpty(bill.originalBillingInvoiceId);
+  (bill.rebillingState === 'rebill' || isEmpty(bill.rebillingState));
```

Comment:
Line 88 could be `['rebill', null].includes(bill.rebillingState)`

Not that it makes a huge difference.